precode.go

```diff
@@ -55,7 +76,18 @@ func main() {
 	var wg sync.WaitGroup
 
 	// 4. Собираем числа из каналов outs
-	// ...
+
+	for i, ch := range outs {
+		wg.Add(1)
+		go func(in <-chan int64, index int) {
+			defer wg.Done()
+
+			for v := range in {
+				chOut <- v
+				atomic.AddInt64(&amounts[index], 1)
```

Comment:
Тут нет гонки данных
```suggestion
				amounts[index]
```